### ISSUE TITLE
Ideas

### ISSUE THREAD

**Issue by agneshraok** [agneshraok/semantic-segmentation-project#2]
# Ideas

* Add images for integer and one hot encoding
* Add bar plot of time taken for training results
* Add bar plot of the size of the model for results
* Try to get the 3d plot of loss function or optimiser.
* Add about dice loss vs categorical cross entropy and say dice is better for imbalanced data
